<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactInfo</class>
 <widget class="QWidget" name="ContactInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="photoLabel">
     <property name="text">
      <string>DEBUG_PHOTO</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>DEBUG_NAME</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>N°Téléphone</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="phoneNumberLabel">
        <property name="text">
         <string>DEBUG_PHONENUMBER</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Adresse Mail</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mailAddressLabel">
        <property name="text">
         <string>DEBUG_MAILADDRESS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Entreprise</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="companyNameLabel">
        <property name="text">
         <string>DEBUG_COMPANYNAME</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>620</width>
        <height>215</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="creationDateLabel">
     <property name="text">
      <string>DEBUG_CREATIONDATE</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
